### PR TITLE
base-images: Latest updates for base images documentation

### PR DESCRIPTION
**Pull request by nghiant2710**
The updates in this PR are:
- Adding new devices information.
- Adding Alpine Linux 3.8
- Adding Ubuntu.

Change-type: patch
Signed-off-by: Trong Nghia Nguyen <nghiant2710@gmail.com>

### CODE REVIEW

File: pages/reference/base-images/base-images.md [hunk 1]
```diff
@@ -657,30 +719,185 @@ __Note:__ In the tree diagram, from the bottom to the top, the lower level image
...
+| {{ $names.company.short }}/kitra520-ubuntu-buildpack-deps | [Docker Hub][armv7hf-kitra520-ubuntu-buildpack-deps-dockerhub-link], [github][armv7hf-kitra520-ubuntu-buildpack-deps-github-link] | latest, jessie, bionic-scm, bionic-curl, xenial, xenial-scm, xenial-curl |
+| {{ $names.company.short }}/kitra520-ubuntu-node | [Docker Hub][armv7hf-kitra520-ubuntu-node-dockerhub-link], [github][armv7hf-kitra520-ubuntu-node-github-link] | For available image tags, refer [here][armv7hf-kitra520-ubuntu-node-dockerhub-tag-link] |
+| {{ $names.company.short }}/kitra520-ubuntu-python | [Docker Hub][armv7hf-kitra520-ubuntu-python-dockerhub-link], [github][armv7hf-kitra520-ubuntu-python-github-link] | For available image tags, refer [here][armv7hf-kitra520-ubuntu-python-dockerhub-tag-link] |
+| {{ $names.company.short }}/kitra520-ubuntu-golang | [Docker Hub][armv7hf-kitra520-ubuntu-golang-dockerhub-link], [github][armv7hf-kitra520-ubuntu-golang-github-link] | For available image tags, refer [here][armv7hf-kitra520-ubuntu-golang-dockerhub-tag-link] |
 
 
-##### Nvidia Jetson Tx2
+##### BananPi-M1+
```
Comment (zwalchuk) [inline pages/reference/base-images/base-images.md]: typo: Banana needs one more 'a'

File: pages/reference/base-images/base-images.md [hunk 2]
```diff
@@ -657,30 +719,185 @@ __Note:__ In the tree diagram, from the bottom to the top, the lower level image
...
+| {{ $names.company.short }}/orangepi-plus2-ubuntu-buildpack-deps | [Docker Hub][armv7hf-orangepi-plus2-ubuntu-buildpack-deps-dockerhub-link], [github][armv7hf-orangepi-plus2-ubuntu-buildpack-deps-github-link] | latest, jessie, bionic-scm, bionic-curl, xenial, xenial-scm, xenial-curl |
+| {{ $names.company.short }}/orangepi-plus2-ubuntu-node | [Docker Hub][armv7hf-orangepi-plus2-ubuntu-node-dockerhub-link], [github][armv7hf-orangepi-plus2-ubuntu-node-github-link] | For available image tags, refer [here][armv7hf-orangepi-plus2-ubuntu-node-dockerhub-tag-link] |
+| {{ $names.company.short }}/orangepi-plus2-ubuntu-python | [Docker Hub][armv7hf-orangepi-plus2-ubuntu-python-dockerhub-link], [github][armv7hf-orangepi-plus2-ubuntu-python-github-link] | For available image tags, refer [here][armv7hf-orangepi-plus2-ubuntu-python-dockerhub-tag-link] |
+| {{ $names.company.short }}/orangepi-plus2-ubuntu-golang | [Docker Hub][armv7hf-orangepi-plus2-ubuntu-golang-dockerhub-link], [github][armv7hf-orangepi-plus2-ubuntu-golang-github-link] | For available image tags, refer [here][armv7hf-orangepi-plus2-ubuntu-golang-dockerhub-tag-link] |
+
+
+##### BALENA FIN
```
Comment (zwalchuk) [inline pages/reference/base-images/base-images.md]: Let's make the capitalization here match the rest, Balena Fin

File: pages/reference/base-images/base-images.md (hunk 1 above)
Comment (nghiant2710) [reply pages/reference/base-images/base-images.md]: fixed

File: pages/reference/base-images/base-images.md (hunk 2 above)
Comment (nghiant2710) [reply pages/reference/base-images/base-images.md]: fixed

**Comment by nghiant2710** [comment]
@zwalchuk, can you check the PR again please?

**Pull request merged**